tests/sqllogictests/suites/base/05_ddl/05_0019_ddl_create_view

```diff
@@ -110,7 +110,7 @@ CREATE VIEW test
 AS
     SELECT *
     FROM
-        's3://testbucket/admin/data/tuple.parquet' (FILES => ('tuple.parquet','test.parquet'),FILE_FORMAT => 'parquet',PATTERN => '.*.parquet', aws_key_id => 'minioadmin', aws_secret_key => 'minioadmin', endpoint_url => 'http://127.0.0.1:9900/')
+        's3://testbucket/admin/data/tuple.parquet' ( FILES => ('tuple.parquet', 'test.parquet'), FILE_FORMAT => 'parquet', PATTERN => '.*.parquet',aws_key_id='minioadmin', aws_secret_key='minioadmin', endpoint_url='http://127.0.0.1:9900/' )
```

Comment:
mixed used of `=>` and `=` inside one `()`?

Reply:
We can add two test case for this(I guess we support both) instead of one:
1. all `=>`
2. all `=`

Reply:
currently, stage option only supports `=>`. `=` shown above is just a display bug that I'm going to fix.